lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -104,6 +105,14 @@
     name: default
     state: absent
     cluster: new_cluster
+
+# Checking whether service is deployed
+- ecs_service:
+    state: deployed
+    name: console-test-service
+    cluster: new_cluster
+    task_definition: new_cluster-task:2"
+    desired_count: 1
```

Comment:
I don't see where in the code `desired_count` is used, as it is retrieved at line 453?